avro2tf/src/main/scala/com/linkedin/avro2tf/configs/TensorizeInConfig.scala

```diff
@@ -23,82 +23,97 @@ object CombinerType extends Enumeration {
  * Case class for hashing info
  *
  * @param hashBucketSize The number of buckets of each hash function
- * @param numHashFunctions The number of hash functions to use
- * @param combinerType Which combiner to use when collision happens
+ * @param numHashFunctions The number of hash functions to use - defaults to 1
+ * @param combiner Which combiner to use when collision happens
  */
 case class HashInfo(
-  hashBucketSize: Int,
-  numHashFunctions: Int = 1,
-  combinerType: CombinerType.CombinerType = CombinerType.SUM
-)
+                     hashBucketSize: Int,
```

Comment:
can you fixed the style here? we use photon-ml style, you can import [this](https://github.com/linkedin/photon-ml/blob/master/Photon%20Style.xml) code style for scala into intellij

Reply:
Thanks. Let me whack everything with the formatter quickly.